bedrock/cms/models/pages.py

```diff
@@ -39,6 +42,10 @@ def serve(self, request):
         return redirect(self.get_parent().get_full_url())
 
 
+class StructuralPage(AbstractStructuralPage):
```

Comment:
I think you suggested this for a reason (something to do with migrations?)

Reply:
Looks like it dates from when you wanted to make a `AboutPage(StructuralPage)` class and it was creating an extra layer of multi-table inheritance - convo is here https://mozilla.slack.com/archives/D02GDP6PAJG/p1720613090788999

Based on this changeset tho, and the instructions, i don't see you using a custom AboutPage after all, so we could back out the changes here -- or leave them in case we do find we want to have other kinds of StructuralPage subclasses

Reply:
Yeah I wasn't sure what was really needed / expected here, happy to revert if we think defining `AboutPage(StructuralPage)` is overkill.